anago

```diff
@@ -608,22 +608,19 @@ prepare_tree () {
   fi
 
   # rev openapi-spec version files on branch for beta tags
+  # and generate docs (not checked in) for official|rc.
```

Comment:
Is this the latest step at which we could generate the docs? Will this cause git versions to appear as `-dirty`?